src/PIL/Image.py

```diff
@@ -1929,11 +1923,23 @@ def save(self, fp, format=None, **params):
 
         if format.upper() not in SAVE:
             init()
+        save_all = False
+        if 'save_all' in params:
+            save_all = params.pop('save_all')
         if save_all:
             save_handler = SAVE_ALL[format.upper()]
         else:
             save_handler = SAVE[format.upper()]
 
+        if params.get('convert_mode'):
+            plugin = sys.modules[save_handler.__module__]
+            convertedIm = self._convert_mode(plugin, params)
```

Comment:
This variable name violates pep8: https://www.python.org/dev/peps/pep-0008/?#function-and-variable-names

Reply:
Okay, I've changed that.